applications/GeoMechanicsApplication/tests/test_benchmark_set_1.py

```diff
@@ -23,13 +23,14 @@ def setUp(self):
     def tearDown(self):
         # Code here will be placed AFTER every test in this TestCase.
         pass
-
+        
```

Comment:
Probably some unintended extra spaces, tabs and newlines in this file.
The intention was only taking the consolidation test out.

Reply:
Thank you. It is not easy to find this.